### PR TITLE
Changing the sections and more

### PR DESCRIPTION
**Pull request by karanphil**
(Here is the installation procedure for the "Other tools".)

This has become a big PR where I moved a lot of sections around. In summary:

- I added a "What's next" page in the "Arriving to the lab" section. It explains what remains to be done and the content of the next sections.
- I renamed "Our tools" by "Tools" and removed the old "Tools" section. It is now the second section.
- I added a "Other tools" page in "Tools".
- I renamed "Intro to" to "Guides" and placed as the third section. I removed  installation steps from these pages and removed the Software page.
- I updated some of the pages like Scilpy and Dipy.

For the moment, most of the "Guides" pages remain the same. Also, I did not change the Git pages, but I renamed the one in "Coding standards in the lab" for less confusion (there might be a better name).

Have a look! (You should build this one locally) @CHrlS98 @elyz081

### CODE REVIEW

**Review by elyz081 — APPROVED** [review]
I think it's great, it became clearer for me especially for Git !

File: source/arriving/whats_next.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        * :ref:`ref_dmri_explorer`
+        * :ref:`ref_dicom_converters`
+        * :ref:`ref_mricrogl`
+
+      | Follow the links and see the installation steps for the tools you want.
+
+    * | **Become familiar with the tools**
+      | The Tools section is a great resource to learn more about all the tools you have installed and more. For instance, it is where you can learn about our multiple :ref:`ref_flow` like :ref:`ref_tractoflow`. Don't forget to go back to this section whenever you forget something about a tool.
+
+    * | **Improve your knowlegde with our guides**
```
Comment (CHrlS98) [inline source/arriving/whats_next.rst:28]: typo knowledge

File: source/arriving/whats_next.rst [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      | The Guides section provides additionnal information on various subjects. It is a useful resource to gather more knowlegde about :ref:`ref_git`, :ref:`ref_python`, :ref:`ref_terminal`, :ref:`ref_bash` or :ref:`ref_anat`, to name a few important ones.
+
+    * | **Start reading about your project**
```
Comment (CHrlS98) [inline source/arriving/whats_next.rst:31]: which makes me think, we might consider having a bibliography management section (I played a bit with this lately so I could do it... after the CDMRI deadline).

File: source/coding/git.rst [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
-Git
-===
+Git good practice
+=================
 
 .. role:: bash(code)
    :language: bash
 
-If you do not know what is Git, you should visit the page :doc:`Intro to Git <../intro_to/explore_git>` to learn about the basic concepts and commands of Git. This page is specifically to learn how we use Git in the lab, what we expect and how you can contribute.
+If you do not know what is Git, you should visit our guide to :ref:`ref_git` to learn about the basic concepts and commands of Git. This page is specifically to learn how we use Git in the lab, what we expect and how you can contribute.
```
Comment (CHrlS98) [inline source/coding/git.rst:9]: then shouldn't this page come before the git guide in the hierarchy? maybe guides could be the absolute last section (or right above `Interesting scholarships/conferences`)?

File: source/tools/others.rst [hunk 4]
```diff
@@ -1,106 +1,108 @@
...
-.. figure:: /images/intro_to_software_fsl.png
-    :scale: 40 %
-    :align: center
+To install it (it might already be installed on a lab computer), follow these instructions depending on your OS, or refer to the `FAQ <https://github.com/imeka/mi-brain/wiki/FAQ#how-to-install>`_.
 
-    FSLeyes is used here to visualize fMRI data.
+    **On Linux**
+        * Download the Linux version of MI-Brain `here <https://github.com/imeka/mi-brain/releases>`_.
+        * In a terminal, extract the .tar.gz release archive using :bash:`tar -xvzf NAME_OF_THE_FILE.tar.gz -C DIRECTORY_TO_PUT_MI_BRAIN`.
+        * Add this to your .bashrc: :bash:`alias MI-Brain="bash DIRECTORY_TO_PUT_MI_BRAIN/MI-Brain.sh"`.
```
Comment (CHrlS98) [inline source/tools/others.rst:35]: I prefer the alias `mibrain`, it's faster :dancers:

**Review by CHrlS98 — COMMENTED** [review]
great work, minor comments here and there

File: source/arriving/whats_next.rst (hunk 2 above)
Comment (karanphil) [reply source/arriving/whats_next.rst:31]: Very good idea!

**Review by CHrlS98 — APPROVED** [review]

**Pull request merged**